src/examples/learn_quant/meaning.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+from learn_quant.quantifier import QuantifierModel, QuantifierUniverse
+from itertools import product, combinations_with_replacement, permutations
+from ultk.language.sampling import powerset
+import random
+import argparse
+
+
+def create_universe(m_size, x_size):
+
+    possible_quantifiers = []
+
+    for x in combinations_with_replacement([0, 1, 2, 3], r=m_size):
```

Comment:
very minor style point: best to use a nicer name even for something like `x`, e.g. `for combination in ...`